src/modules/crud.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars */
+const containerElement = document.querySelector('.containerTodo');
+const ulElement = document.querySelector('.ulElement');
+
+const Task = class {
+  constructor(description, completed = false, index = 0) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+    this.tasks = [
+      {
+        description: 'wash the dogs',
+        completed: false,
+        index: 0,
+      },
+      {
+        description: 'Complete To Do list project',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'Comwqsaect',
+        completed: false,
+        index: 2,
+      },
+    ];
```

Comment:
- [ Required ] Please consider removing all hardcoded tasks from `this.task` variable, this will ensure that users using your App online have an empty `to-do list` when they start using your application. Currently, the `to-do list` has default tasks which might confuse someone using your App, please rectify this.

## Project requirements
![Screenshot from 2023-01-26 00-45-44](https://user-images.githubusercontent.com/16392046/214698718-659391cb-5e90-459c-a49a-c0756a073fc9.png)